.travis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       cache: npm
 
       node_js:
-        - "8"
+        - "10"
```

Comment:
NP: If there is only one element you can remove list and use standard yaml field